<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGeneticAlgorithm</class>
 <widget class="QDialog" name="DialogGeneticAlgorithm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настрока генетического алгоритма</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/iconSetting.png</normaloff>:/iconSetting.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Кол-во популяций</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_countPopulation">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Кол-во особей в популяции</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_countGenes">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_prematureTermination">
       <property name="text">
        <string>Преждевременное завершение</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Шанс мутации</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_changeMutation">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_ok">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b_cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogGeneticAlgorithm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
